### DISCUSSION TITLE
Use Pagefind with lots of non-matching text

### DISCUSSION THREAD

**Discussion by kenmorse** [Pagefind/pagefind#150] [Q&A] [answered]
I'll just start by saying what a wonderful and useful library Pagefind is.

One additional scenario I think it could be useful for (in addition to the standard "search" at the top of the site) is this:

When people enter more free-form text in an input field or textarea, Pagefind won't always return results. For example, enter this in the search on the [Pagefind page](https://pagefind.app/):

    I need help with pagefind and hugo along with jekyll

and you'll see 

    No results for I need help with pagefind and hugo along with jekyll

Even though many of those words are indeed on the site, no results are returned.

It may just be that Pagefind isn't currently designed for a search like this, but I'd be interested to find a way to try to make it work.

**Comment by bglw** [comment, marked as answer]
👋 Thanks for the kind words!

You're correct that this is currently just an omission in the internal design — the primary goal for release was bandwidth reduction, so it hasn't yet seen a big focus on the search experience.

This is coming though! Alongside improvements to result ranking using word distances and other niceties. I'll keep you posted on when that's closer to coming to fruition 🙂
